docs/configuration-process.asciidoc

```diff
@@ -10,11 +10,27 @@ apm-server:
   rum:
     enabled: true
 
+output:
+  elasticsearch:
+    hosts: ElasticsearchAddress:9200
```

Comment:
It would be more consistent to align this with [L29](https://github.com/elastic/apm-server/pull/2170/files#diff-7b74f374359d31b837afa311b6f055efR29) ` hosts: ["ElasticsearchAddress:9200"]`.